tests/test_algorithm.py

```diff
@@ -214,16 +195,16 @@ def test_record_incr(self):
                                       range(1, len(output) + 1))
 
 
-class TestMiscellaneousAPI(WithLogger,
-                           WithSimParams,
-                           WithDataPortal,
-                           ZiplineTestCase):
+class TestMiscellaneousAPI(zf.WithMakeAlgo, zf.ZiplineTestCase):
 
     START_DATE = pd.Timestamp('2006-01-03', tz='UTC')
     END_DATE = pd.Timestamp('2006-01-04', tz='UTC')
     SIM_PARAMS_DATA_FREQUENCY = 'minute'
     sids = 1, 2
 
+    # FIXME: Pass a benchmark source instead of this.
```

Comment:
The default behavior of `WithMakeAlgo` is to use the first equity sid as a benchmark, but that often doesn't work, because if you run an algorithm from START_DATE to END_DATE, then you don't have enough days of historical data to calculate benchmark returns (in particular, you need the close from the day before START_DATE to compute a daily benchmark return for the first day of the backtest.

Setting BENCHMARK_SID to None causes us to use benchmark returns retrieved from the TradingEnvironment, which uses a static set of data that's checked into git.  I'm not sure what we want long-term for benchmarks in testing, but this is what we were already doing in all of these tests, and I didn't want to layer in more changes than were necessary.  Once we have all these suites creating algorithms in a uniform way, it will be easier to make broad changes to how we handle benchmarks.